packages/web-components/src/components/content-item-horizontal/__stories__/README.stories.mdx

```diff
@@ -60,6 +60,25 @@ import '@carbon/ibmdotcom-web-components/es/components/content-item-horizontal/i
 </dds-content-item-horizontal>
 ```
 
+### HTML (with thumbnail)
+
+```html
+<dds-content-item-horizontal thumbnail="true">
+  <dds-content-item-heading>Aliquam condimentum</dds-content-item-heading>
+  <dds-content-item-horizontal-copy>Lorem ipsum dolor sit amet, _consectetur_ adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit.
+    Phasellus at elit sollicitudin.</dds-content-item-horizontal-copy>
+  <dds-link-list slot="footer" type="vertical">
+    <dds-link-list-item-cta icon-placement="right" href="https://www.ibm.com" cta-type="local">
+      Learn more
+    </dds-link-list-item-cta>
+    <dds-link-list-item-cta icon-placement="right" href="https://www.ibm.com" cta-type="external">
+      Microservices and containers
+    </dds-link-list-item-cta>
+  </dds-link-list>
+  <dds-image slot="thumbnail" alt="Alt text" default-src="image.jpg"></dds-image>
```

Comment:
```suggestion
  <dds-image slot="thumbnail" alt="Alt text" default-src="https://fpoimg.com/960x720?text=4:3&bg_color=ee5396&text_color=161616"></dds-image>
```

Let's use FPO image for easy copy and paste of this WC markup so that adopters can see what this variant will look like instead of defending the URL of the image.